### DISCUSSION TITLE
Allow customizing the delimiter for the query builder's `in` filter

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5648] [Feature Requests]
Similar to #2190, but for the query builder

**Comment by grantfitzsimmons** [comment]
What happens when you search locality names when using the `in` filter?

Say you had two names:
`Lawrence, Kansas, United States`, `Topeka, Kansas, United States`

If you put a single quote (`'`) around them, would it work correctly when using the filter? If so, I think we should close this.

**Comment by maxpatiiuk** [comment]
I don't think it will work correctly.
Though, I also don't think we should be fixing this issue until sp6 is retired as it would introduce even more incompatibilities between the queries in different specify versions

**Comment by maxpatiiuk** [comment]
Another though about this:
It would probably be a good idea for the delimiter to be stored on the query (need to add a new field to the table), rather than in some sort of app resource (the default value could still come from an app resource though).
Think of it:
- if someone changes the delimiter, all previous queries would break (or we would have to add extra logic to update all queries) - but if queries have their own delimiter field, old queries would keep on using old delimiter and new queries will use new delimiter
- if exporting/importing queries between collections or databases, which may have different delimiters, things would break, unless delimiter is a field on the query
- once query sharing becomes a thing, depending on how it's implemented, the default query delimiter for current user might not match the default delimiter of the user who created the query (i.e, if the pref is coming from differently scoped app resources)
- Adding a new field on the table would also provide backwards compatibility - if query has a delimiter field, use it, otherwise assume query is from an older version of specify, and so default to using ","
